packages/mobile/src/onboarding/success/OnboardingSuccessScreen.tsx

```diff
@@ -0,0 +1,28 @@
+import colors from '@celo/react-components/styles/colors'
+import React, { useCallback } from 'react'
+import { StyleSheet, View } from 'react-native'
+import DancingRings from 'src/icons/DancingRings'
+import { navigateHome } from 'src/navigator/NavigationService'
+
+function OnboardingSuccessScreen() {
+  const onAnimationFinish = useCallback(() => {
+    navigateHome()
+  }, [])
+
+  return (
+    <View style={styles.container}>
+      <DancingRings onAnimationFinish={onAnimationFinish} />
```

Comment:
nit: `navigateHome` could be passed directly.

Reply:
Good idea. We used to do other things in that callback.